ui/pages/routes/modals.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from '../../store/store';
+import { setEditedNetwork } from '../../store/actions';
+import { NetworkListMenu } from '../../components/multichain/network-list-menu';
+import { useModalState } from '../../hooks/useModalState';
+
+export const Modals = () => {
```

Comment:
Setting up the foundation for consolidating the modals

Currently only contains Network menu, extracted out of routes.component.tsx